tests/test_time_series/test_forecasting/test_expanding_window_features.py

```diff
@@ -550,3 +550,20 @@ def test_transform_x_y(df_time):
     assert len(Xt) == len(yt)
     assert len(y) != len(yt)
     assert (Xt.index == yt.index).all()
+
+
+def test_error_duplicate_functions(df_time):
+    with pytest.raises(ValueError):
+        ExpandingWindowFeatures(
+            variables=["ambient_temp"], functions=["sum", "sum"], periods=2, freq="15T"
+        )
```

Comment:
is this an error that is raised from within feature engine? if yes, we'd also need to assert the error message is the expected one

Reply:
Done